EpsilonAuxBmsV2/Tests/tests/TripTest/TripTest.c

```diff
@@ -0,0 +1,134 @@
+#include "unity.h"
+#include "TripTest.h"
+#include "Trip.h"
+#include "Mockstm32f4xx_hal_gpio.h"
+
+void runTripTests()
+{
+    RUN_TEST(test_checkDischargeTripDueToLowCell);
+    RUN_TEST(test_checkDischargeTripDueToHighTempAndCurrent);
+    RUN_TEST(test_checkDischargeTripDueToPackCurrent);
+    RUN_TEST(test_checkDischargeTripForNoTrip);
+    RUN_TEST(test_checkChargeTripDueToHighCell);
+    RUN_TEST(test_checkChargeTripDueToHighTempAndCurrent);
+    RUN_TEST(test_checkChargeTripDueToPackCurrent);
+    RUN_TEST(test_checkChargeTripForNoTrip);
+    RUN_TEST(test_checkProtectionTripForTrip);
+    RUN_TEST(test_checkProtectionTripForNoTrip);
+}
+
+void test_checkDischargeTripDueToLowCell()
+{
+    OrionCanInfo message = {0};
+    AuxTrip auxTripToUpdate = {0};
```

Comment:
I would suggest making these global variables and only setting the `message` as necessary. This will make sure that state carried between the tests doesn't affect things